nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/xml/XMLRecordSetWriter.java

```diff
@@ -80,6 +80,16 @@
             .required(true)
             .build();
 
+    public static final PropertyDescriptor OMIT_XML_DECLARATION = new PropertyDescriptor.Builder()
+            .name("omit_xml_declaration")
+            .displayName("Omit XML Declaration")
+            .description("Specifies whether or not to include XML declaration")
+            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
+            .allowableValues("true", "false")
+            .defaultValue("false")
+            .required(false)
```

Comment:
The standard convention for properties with a set of allowable values is to make the property required. Functionally it doesn't matter but I will set it to true for consistency when I merge.

Reply:
Sure